From 49d4bc6c02cd579eca4659af2904ea2ce80ec3b7 Mon Sep 17 00:00:00 2001
From: Mark Harfouche <mark.harfouche@gmail.com>
Date: Sun, 6 Jul 2025 16:27:57 -0400
Subject: [PATCH 1/2] DO not link to proto_api

---
 pybind11_protobuf/BUILD | 1 -
 1 file changed, 1 deletion(-)

diff --git a/pybind11_protobuf/BUILD b/pybind11_protobuf/BUILD
index d9abcaa..3393167 100644
--- a/pybind11_protobuf/BUILD
+++ b/pybind11_protobuf/BUILD
@@ -46,7 +46,6 @@ pybind_library(
         "@com_google_absl//absl/strings",
         "@com_google_absl//absl/types:optional",
         "@com_google_protobuf//:protobuf",
-        "@com_google_protobuf//python:proto_api",
     ],
 )
 
-- 
2.43.0

