From 412a80c0e94daa47d9206aef26b1d13af9dc84ef Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Tue, 16 Jul 2024 17:18:23 +0200
Subject: [PATCH 21/31] Only link to sparse_core_layout_proto_cc headers

---
 tensorflow/core/tpu/kernels/BUILD | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tensorflow/core/tpu/kernels/BUILD b/tensorflow/core/tpu/kernels/BUILD
index 94bf252e09e..352329dde93 100644
--- a/tensorflow/core/tpu/kernels/BUILD
+++ b/tensorflow/core/tpu/kernels/BUILD
@@ -1533,7 +1533,7 @@ cc_library(
     hdrs = ["sparse_core_layout.h"],
     visibility = ["//tensorflow/python/tpu:__pkg__"],  # ONLY for `_pywrap_sparse_core_layout`.
     deps = [
-        ":sparse_core_layout_proto_cc",
+        ":sparse_core_layout_proto_cc_headers_only",
         "//tensorflow/core/platform:stringpiece",
         "@com_google_absl//absl/container:btree",
         "@com_google_absl//absl/log:check",
